mozilla-release/other-licenses/nsis/Contrib/CliqzHelper/dllmain.cpp

```diff
@@ -0,0 +1,76 @@
+#include <windows.h>
+#include <shlobj.h>
+#include <stdio.h>
```

Comment:
I'd sort headers by name and also separate standard library headers from system ones,